pyanaconda/modules/payload/live/live_os.py

```diff
@@ -100,3 +122,55 @@ def teardown_installation_source_with_task(self):
         task = TeardownInstallationSourceTask(INSTALL_TREE)
 
         return self.publish_task(LIVE_OS_HANDLER.namespace, task)
+
+    def pre_install_with_task(self):
+        """Prepare intallation task."""
+        task = PrepareSystemForInstallation()
+
+        return self.publish_task(LIVE_OS_HANDLER.namespace, task)
+
+    def install_with_task(self):
+        """Install the payload."""
+        task = InstallFromImageTask(
+            getSysroot(),
+            self.kernel_version_list
+        )
+        return self.publish_task(LIVE_OS_HANDLER.namespace, task)
+
+    def post_install_with_task(self):
```

Comment:
Just to be sure, this one (and API method) should be renamed to post_install_with_tasks as well.

Reply:
Should be done by the newest update.